### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_Confirm Selected Features_Screen Reader: Narrator is reading the status of the Azure toolkit checkbox as ‘Checked’ instead of reading ‘Expanded’ or ‘Collapsed

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3353]
[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #C+E-H; #Java tooling for IntelliJ/Eclipse; #Narrator

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: Narrator
MAS Violated: MAS1.3.1

**Repro Steps:** 
1) Start narrator and Open Java EE - Eclipse application
2) Tab till Help-->Eclipse Marketplace
3) Find for 'Azure Toolkit for Eclipse' and press enter
4) Tab till Install now button and press enter
5) Tab till Azure Toolkit checkbox and expand or collapse the tree view list items

**Actual Result:** 
Narrator is reading the status of the Azure toolkit checkbox as ‘Checked’ instead of reading ‘Expanded’ or ‘Collapsed

**Expected Result:** 
When expanded or collapsed Narrator must read the status as ‘Expanded’ or ‘Collapsed’ instead of checked

**User Impact:** 
Screen Reader users find it difficult to understand the status if Narrator is reading the expand/Collapse status wrong

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_23_Narrator is not reading expand collapse status.pptx](https://github.com/microsoft/azure-tools-for-java/files/3455422/Bug_23_Narrator.is.not.reading.expand.collapse.status.pptx)

**Comment by Flanker32** [comment]
Hi @soumyatippapuram, The marketplace UI is controlled by eclipse, which is out control of our plugin,  so could you please open an issue in eclipse(https://bugs.eclipse.org/bugs/). We could verify this issue by search AWS Toolkits in Eclipse marketplace and we could find the same behavior of this issue.
Besides, NVDA could read the Expanded/Collapsed status correctly,

**Comment by Flanker32** [comment]
- Importance of experience: Little, it's not a common behavior to check the install items while install plugins, we hadn't received customer feedback about this issue.
- Quality of experience: Little, users could use NVDA to get the correct Expanded/Collapsed status
- Alternate solutions: Users could use NVDA to get the correct Expanded/Collapsed status
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Tanushagottipati This is an external issue of Eclipse which has been linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us review it, thanks a lot!

**Comment by Tanushagottipati** [comment]
External Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=551115

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749753
